packages/vaadin-virtual-list/src/virtualizer-iron-list-adapter.js

```diff
@@ -417,6 +417,13 @@ export class IronListAdapter {
         this.elementsContainer.insertBefore(visibleElements[i], visibleElements[0]);
       }
     }
+
+    // Due to a rendering bug, reordering the rows can make parts of the scroll target disappear
+    // on Safari when using sticky positioning in case the scroll target is inside a flexbox.
+    // This issue manifests with grid (the header can disappear if grid is used inside a flexbox)
+    const { transform } = this.scrollTarget.style;
```

Comment:
Would it work if we used this workaround only for Safari? 

I wonder if moving an element to a new layer and back can be a quite heavy operation.

Reply:
Done